src/autobridgecalculator.js

```diff
@@ -70,14 +73,17 @@ AutobridgeCalculator.prototype.calculate = function() {
       continue;
     }
 
-    if (legOneTakerGetsFunded.compareTo(legTwoTakerPaysFunded) > 0) {
+    //  using private fields for speed
```

Comment:
Other languages have notion of "friends" classes, specially for this case.
`AutobridgeCalculator` and `OrderBook` are internal to ripplelib code, so I think they are allowed to use some private parts of other classes.
I think we can create something like  `Amount._createFast` that encapsulate fast way of creation `Amount` object.